program_transformers/src/token/mod.rs

```diff
@@ -116,15 +116,16 @@ pub async fn handle_token_program_account<'a, 'b>(
             db.execute(query).await?;
 
             let txn = db.begin().await?;
+            let is_non_fungible = m.decimals == 0 && m.mint_authority.is_none();
```

Comment:
m.is_fungible()

Extend the interface of the model to include an is_fungible method.